### PR TITLE
Name retagging

### PR DESCRIPTION
**Pull request by nschneid**
Limits PROPN to true nouns, even in names (#164)

### CODE REVIEW

File: not-to-release/sources/answers/20111106015552AAj6rCu_ans.xml.conllu [hunk 1]
```diff
@@ -17,7 +17,7 @@
 13	from	from	ADP	IN	_	7	obl	7:obl	_
 14	in	in	ADP	IN	_	15	case	15:case	_
 15	Invercargill	Invercargill	PROPN	NNP	Number=Sing	1	obl	1:obl:in	_
-16	New	New	PROPN	NNP	Number=Sing	17	compound	17:compound	_
+16	New	New	ADJ	NNP	Degree=Pos	17	amod	17:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/answers/20111106015552AAj6rCu_ans.xml.conllu:20]: NB in GUM New Zealand and New York are PROPN + flat, based on the idea that New is now completely opaque and it's just a flat name. Would you be OK adopting that?

File: not-to-release/sources/answers/20111107180248AAnQ3aE_ans.xml.conllu [hunk 2]
```diff
@@ -8,7 +8,7 @@
 4	can	can	AUX	MD	VerbForm=Fin	6	aux	6:aux	_
 5	I	I	PRON	PRP	Case=Nom|Number=Sing|Person=1|PronType=Prs	6	nsubj	6:nsubj	_
 6	watch	watch	VERB	VB	VerbForm=Inf	0	root	0:root	_
-7	Fair	Fair	PROPN	NNP	Number=Sing	8	compound	8:compound	_
+7	Fair	Fair	ADJ	NNP	Degree=Pos	8	amod	8:amod	_
 8	City	City	PROPN	NNP	Number=Sing	6	obj	6:obj	_
 9	Online	online	ADV	RB	_	6	advmod	6:advmod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/answers/20111107180248AAnQ3aE_ans.xml.conllu:13]: Shouldn't Online be NNP/Online too?

File: not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu [hunk 3]
```diff
@@ -5,7 +5,7 @@
 1	Any	any	DET	DT	_	2	det	2:det	_
 2	information	information	NOUN	NN	Number=Sing	0	root	0:root	_
 3	about	about	ADP	IN	_	6	case	6:case	_
-4	CRAZY	CRAZY	PROPN	NNP	Number=Sing	5	compound	5:compound	_
+4	CRAZY	CRAZY	ADJ	NNP	Degree=Pos	5	amod	5:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu:8]: Lemmas should have title case?

File: not-to-release/sources/answers/20111108084119AAhsBk8_ans.xml.conllu [hunk 4]
```diff
@@ -143,7 +143,7 @@
 9	bugless	bugless	PROPN	NNP	Number=Sing	10	compound	10:compound	_
 10	beast	beast	PROPN	NNP	Number=Sing	7	nmod	7:nmod:like	_
 11	and	and	CCONJ	CC	_	13	cc	13:cc	_
-12	baked	baked	PROPN	NNP	Number=Sing	13	compound	13:compound	_
+12	baked	bake	VERB	NNP	Tense=Past|VerbForm=Part	13	amod	13:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/answers/20111108084119AAhsBk8_ans.xml.conllu:146]: If it's NNP shouldn't lemma be cap?

File: not-to-release/sources/answers/20111108101024AANYJpw_ans.xml.conllu [hunk 5]
```diff
@@ -170,7 +170,7 @@
 5	SUCH	such	ADJ	JJ	Degree=Pos	6	amod	6:amod	_
 6	THING	thing	NOUN	NN	Number=Sing	3	nsubj	3:nsubj	_
 7	AS	as	ADP	IN	_	10	case	10:case	_
-8	BASIC	BASIC	PROPN	NNP	Number=Sing	10	compound	10:compound	_
+8	BASIC	BASIC	ADJ	NNP	Degree=Pos	10	amod	10:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/answers/20111108101024AANYJpw_ans.xml.conllu:173]: Title case lemma

File: not-to-release/sources/answers/20111107180248AAnQ3aE_ans.xml.conllu (hunk 2 above)
Comment (nschneid) [reply not-to-release/sources/answers/20111107180248AAnQ3aE_ans.xml.conllu:13]: I don't think it's part of the name.

File: not-to-release/sources/email/enronsent32_01.xml.conllu [hunk 6]
```diff
@@ -334,10 +334,10 @@
 19	obligations	obligation	NOUN	NNS	Number=Plur	15	nmod	15:nmod:of	_
 20	under	under	ADP	IN	_	28	case	28:case	_
 21	the	the	DET	DT	Definite=Def|PronType=Art	28	det	28:det	_
-22	Indian	Indian	PROPN	NNP	Number=Sing	24	compound	24:compound	_
+22	Indian	Indian	ADJ	NNP	Degree=Pos	24	amod	24:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/email/enronsent32_01.xml.conllu:337]: This tree looks messed up?

File: not-to-release/sources/email/enronsent35_01.xml.conllu [hunk 7]
```diff
@@ -1904,7 +1904,7 @@
 7	guidelines	guideline	NOUN	NNS	Number=Plur	5	obj	5:obj	_
 8	for	for	ADP	IN	_	12	case	12:case	_
 9	the	the	DET	DT	Definite=Def|PronType=Art	12	det	12:det	_
-10	Southern	Southern	PROPN	NNP	Number=Sing	11	compound	11:compound	_
+10	Southern	Southern	ADJ	NNP	Degree=Pos	11	amod	11:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/email/enronsent35_01.xml.conllu:1907]: +1 for Southern as amod and South as compound, doing the same in GUM

File: not-to-release/sources/email/enronsent17_01.xml.conllu [hunk 8]
```diff
@@ -207,14 +207,14 @@
 # text = 5. General Colin Powell, former Chairman, Joint Chiefs of Staff, U.S.A.
 1	5	5	NUM	LS	_	4	nummod	4:nummod	SpaceAfter=No
 2	.	.	PUNCT	.	_	1	punct	1:punct	_
-3	General	General	PROPN	NNP	Number=Sing	4	compound	4:compound	_
+3	General	General	ADJ	NNP	Degree=Pos	4	amod	4:amod	_
```
Comment (nschneid) [inline not-to-release/sources/email/enronsent17_01.xml.conllu:210]: need to be careful about General

**Review by amir-zeldes — COMMENTED** [review]
Oh, weird, I wonder why they capitalized Online then.

File: not-to-release/sources/answers/20111106015552AAj6rCu_ans.xml.conllu (hunk 1 above)
Comment (nschneid) [reply not-to-release/sources/answers/20111106015552AAj6rCu_ans.xml.conllu:20]: I don't like having to make a semantic distinction. Is General Motors semantically compositional? Good Friday? Old Faithful? Seems like a can of worms.

File: not-to-release/sources/newsgroup/groups.google.com_CaptSpasticsJokes_4c06327c7fb39291_ENG_20050727_125600.xml.conllu [hunk 9]
```diff
@@ -49,7 +49,7 @@
 2-3	Spastic's	_	_	_	_	_	_	_	_
 2	Spastic	Spastic	PROPN	NNP	Number=Sing	5	nmod:poss	5:nmod:poss	_
 3	's	's	PART	POS	_	2	case	2:case	_
-4	Demented	Demented	PROPN	NNP	Number=Sing	5	compound	5:compound	_
+4	Demented	Dement	VERB	NNP	Tense=Past|VerbForm=Part	5	amod	5:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/newsgroup/groups.google.com_CaptSpasticsJokes_4c06327c7fb39291_ENG_20050727_125600.xml.conllu:52]: Not sure this is really VERB

File: not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu (hunk 3 above)
Comment (nschneid) [reply not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu:8]: No I backed off of changing the case in the lemma

File: not-to-release/sources/newsgroup/groups.google.com_FOOLED_7554c5ce34a5a49e_ENG_20051012_144800.xml.conllu [hunk 10]
```diff
@@ -1201,7 +1201,7 @@
 16	rocket	rocket	NOUN	NN	Number=Sing	9	nmod	9:nmod:of	SpaceAfter=No
 17	,	,	PUNCT	,	_	16	punct	16:punct	_
 18	the	the	DET	DT	Definite=Def|PronType=Art	21	det	21:det	_
-19	Long	Long	PROPN	NNP	Number=Sing	21	compound	21:compound	_
+19	Long	Long	ADJ	NNP	Degree=Pos	21	amod	21:amod	_
 20	March	March	PROPN	NNP	Number=Sing	21	compound	21:compound	_
 21	V	V	PROPN	NNP	Number=Sing	16	appos	16:appos	SpaceAfter=No
```
Comment (amir-zeldes) [inline not-to-release/sources/newsgroup/groups.google.com_FOOLED_7554c5ce34a5a49e_ENG_20051012_144800.xml.conllu:1206]: Is "V" really the head?

File: not-to-release/sources/newsgroup/groups.google.com_GayMarriage_0ccbb50b41a5830b_ENG_20050321_181500.xml.conllu [hunk 11]
```diff
@@ -4,7 +4,7 @@
 # text = NORTH CAROLINA RELIGIOUS COALITION FOR MARRIAGE EQUALITY
 1	NORTH	NORTH	PROPN	NNP	Number=Sing	2	compound	2:compound	_
 2	CAROLINA	CAROLINA	PROPN	NNP	Number=Sing	4	compound	4:compound	_
-3	RELIGIOUS	RELIGIOUS	PROPN	NNP	Number=Sing	4	compound	4:compound	_
+3	RELIGIOUS	RELIGIOUS	ADJ	NNP	Degree=Pos	4	amod	4:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/newsgroup/groups.google.com_GayMarriage_0ccbb50b41a5830b_ENG_20050321_181500.xml.conllu:7]: Title case

File: not-to-release/sources/email/enronsent32_01.xml.conllu (hunk 6 above)
Comment (nschneid) [reply not-to-release/sources/email/enronsent32_01.xml.conllu:337]: I think Roman numerals in non-flat names are treated as heads right now (e.g. "World War II"). Not ideal but it's a separate issue.

File: not-to-release/sources/newsgroup/groups.google.com_alt.animals_0e65f540816d780c_ENG_20041116_124800.xml.conllu [hunk 12]
```diff
@@ -19,7 +19,7 @@
 
 # sent_id = newsgroup-groups.google.com_alt.animals_0e65f540816d780c_ENG_20041116_124800-0003
 # text = UNITED PRESS INTERNATIONAL
-1	UNITED	UNITED	PROPN	NNP	Number=Sing	2	compound	2:compound	_
+1	UNITED	UNITED	ADJ	NNP	Degree=Pos	2	amod	2:amod	_
 2	PRESS	PRESS	PROPN	NNP	Number=Sing	3	compound	3:compound	_
```
Comment (amir-zeldes) [inline not-to-release/sources/newsgroup/groups.google.com_alt.animals_0e65f540816d780c_ENG_20041116_124800.xml.conllu:23]: Title case - will stop commenting on these, probably easiest to find automatically

**Review by amir-zeldes — COMMENTED** [review]
Definitely not changing Roman-numeral-containing names to be headed by the numeral in GUM.... That seems like a downgrade. Not sure about opaque "New", it's been kind of a staple example of `flat` in place names for us, so I think it would need more discussion before changing in GUM.

File: not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu (hunk 3 above)
Comment (amir-zeldes) [reply not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu:8]: OK, for the record these are all title-cased when appropriate in GUM

File: not-to-release/sources/newsgroup/groups.google.com_CaptSpasticsJokes_4c06327c7fb39291_ENG_20050727_125600.xml.conllu (hunk 9 above)
Comment (nschneid) [reply not-to-release/sources/newsgroup/groups.google.com_CaptSpasticsJokes_4c06327c7fb39291_ENG_20050727_125600.xml.conllu:52]: Happy to change it to ADJ (though I don't think EWT is particularly consistent about the VERB/ADJ boundary)

File: not-to-release/sources/newsgroup/groups.google.com_hiddennook_04d8cc994875d454_ENG_20050207_012300.xml.conllu [hunk 13]
```diff
@@ -503,10 +503,10 @@
 # text = Posted by Hidden Nook to Hidden Nook at 2/7/2005 01:09:32 AM
 1	Posted	post	VERB	VBN	Tense=Past|VerbForm=Part	0	root	0:root	_
 2	by	by	ADP	IN	_	4	case	4:case	_
-3	Hidden	Hidden	PROPN	NNP	Number=Sing	4	compound	4:compound	_
+3	Hidden	Hidden	ADJ	NNP	Degree=Pos	4	amod	4:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/newsgroup/groups.google.com_hiddennook_04d8cc994875d454_ENG_20050207_012300.xml.conllu:506]: VERB?

File: not-to-release/sources/newsgroup/groups.google.com_hiddennook_88969236563fa748_ENG_20050215_173600.xml.conllu [hunk 14]
```diff
@@ -243,7 +243,7 @@
 18	pleased	pleased	ADJ	JJ	Degree=Pos	17	xcomp	17:xcomp	_
 19	that	that	SCONJ	IN	_	24	mark	24:mark	_
 20	the	the	DET	DT	Definite=Def|PronType=Art	22	det	22:det	_
-21	White	White	PROPN	NNP	Number=Sing	22	compound	22:compound	_
+21	White	White	ADJ	NNP	Degree=Pos	22	amod	22:amod	_
```
Comment (amir-zeldes) [inline not-to-release/sources/newsgroup/groups.google.com_hiddennook_88969236563fa748_ENG_20050215_173600.xml.conllu:246]: Not necessarily saying we should change this, but oddly "White House" is sometimes given as a paradigm example of an English A+N compound, based on the Compound Stress Rule (cf. "Some white 'house")

File: not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu (hunk 3 above)
Comment (nschneid) [reply not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu:8]: Yeah, I think eventually we should go through all the NNPs and make the casing consistent. But doing it now for just the non-PROPN NNPs would make things more confusing I think.

File: not-to-release/sources/reviews/203196.xml.conllu [hunk 15]
```diff
@@ -8,7 +8,7 @@
 # newpar id = reviews-203196-p0002
 # text = VISAKHA INDUSTRIAL GASES PVT. LTD., location at google maps.
 1	VISAKHA	VISAKHA	PROPN	NNP	Number=Sing	5	compound	5:compound	_
-2	INDUSTRIAL	INDUSTRIAL	PROPN	NNP	Number=Sing	5	compound	5:compound	_
+2	INDUSTRIAL	INDUSTRIAL	ADJ	NNP	Degree=Pos	5	amod	5:amod	_
 3	GASES	GASES	PROPN	NNPS	Number=Plur	5	compound	5:compound	_
 4	PVT.	PVT.	PROPN	NNP	Number=Sing	5	compound	5:compound	_
 5	LTD.	LTD.	PROPN	NNP	Number=Sing	0	root	0:root	SpaceAfter=No
```
Comment (amir-zeldes) [inline not-to-release/sources/reviews/203196.xml.conllu:14]: In the same bucket as Roman numerals as heads, LTD. as head looks off to me

**Review by amir-zeldes — COMMENTED** [review]
Overall this is a huge improvement IMO, great work!

File: not-to-release/sources/email/enronsent32_01.xml.conllu (hunk 6 above)
Comment (nschneid) [reply not-to-release/sources/email/enronsent32_01.xml.conllu:337]: #167

File: not-to-release/sources/newsgroup/groups.google.com_FOOLED_7554c5ce34a5a49e_ENG_20051012_144800.xml.conllu (hunk 10 above)
Comment (nschneid) [reply not-to-release/sources/newsgroup/groups.google.com_FOOLED_7554c5ce34a5a49e_ENG_20051012_144800.xml.conllu:1206]: #167

File: not-to-release/sources/reviews/203196.xml.conllu (hunk 15 above)
Comment (nschneid) [reply not-to-release/sources/reviews/203196.xml.conllu:14]: #167

File: not-to-release/sources/newsgroup/groups.google.com_hiddennook_04d8cc994875d454_ENG_20050207_012300.xml.conllu (hunk 13 above)
Comment (nschneid) [reply not-to-release/sources/newsgroup/groups.google.com_hiddennook_04d8cc994875d454_ENG_20050207_012300.xml.conllu:506]: It's more often ADJ elsewhere in the corpus, but the line is not consistent. See #102.

File: not-to-release/sources/newsgroup/groups.google.com_hiddennook_88969236563fa748_ENG_20050215_173600.xml.conllu (hunk 14 above)
Comment (nschneid) [reply not-to-release/sources/newsgroup/groups.google.com_hiddennook_88969236563fa748_ENG_20050215_173600.xml.conllu:246]: Yeah we definitely are not defining `compound` to include adjective-noun compounds.

**Pull request merged**